.pre-commit-config.yaml

```diff
@@ -3,14 +3,11 @@ repos:
     rev: 22.10.0
     hooks:
     -   id: black
-        args: [ "--force-exclude=bastionai" ]
-        exclude: bastionai/
-    -   id: black-jupyter
-        args: [ "--force-exclude=bastionai" ]
+        args: [ "--check", "--force-exclude=bastionai" ]
```

Comment:
Why do you change the CI?

Reply:
I... didn't ? 
Not sure what happened there.